oak-blob-cloud-azure/src/main/java/org/apache/jackrabbit/oak/blob/cloud/azure/blobstorage/AzureDataStore.java

```diff
@@ -41,11 +42,18 @@ public class AzureDataStore extends AbstractSharedCachingDataStore implements Co
 
     protected Properties properties;
 
-    private AzureBlobStoreBackend azureBlobStoreBackend;
+    private AbstractAzureBlobStoreBackend azureBlobStoreBackend;
+
+    private final boolean useAzureSdkV12 = Boolean.getBoolean("blob.azure.v12.enabled");
```

Comment:
If the choice of the SDK depends on a system property, how do we get test coverage for the old SDK?

Reply:
I need to check how the original author tested that.